doc/api/v2/fluid/initializer.rst

```diff
@@ -0,0 +1,50 @@
+===========
+Initializer
+===========
+
+
+
+Initializer
+-----------
+..  automodule:: paddle.v2.fluid.initializer
+    :members: Initializer
+    :noindex:
+
+
+
+ConstantInitializer
+-----------
```

Comment:
line 16 should be longer than line 15, otherwise, there will be warnings when compile paddle_docs

Reply:
Done